### ISSUE TITLE
Add status code 410 to abort redner()

### ISSUE THREAD

**Issue by bagr001** [brillout/vite-plugin-ssr#1097]
### Description

Hi, could you please add error code 410 to the abort redner()? Thanks.

```
// Render the error page
render(statusCode: 401 | 403 | 404 | 410 | 429 | 500 | 503, abortReason?: string)
```

**Comment by brillout** [comment]
Could you use 404 instead? What's the (practical) difference between 404 and 410?

**Comment by bagr001** [comment]
Unfortunately not. 410 is necessary for good SEO as there is a semantic difference.

> The 404 indicates, that the resource is not present at the given location, and it has probably never been (or server has no idea whether it has been).

> The 410, on the other hand, indicates that resource is not present anymore, but it used to be there in the past. It's a useful hint for some clients such as search engines, spiders etc., because they can remove this resource from their indexes.

**Comment by brillout** [comment]
Pre-released as `0.4.139-commit-ac52a24`.

**Comment by brillout** [comment]
I probably told you this already, but in case your company is up for it: https://github.com/sponsors/brillout.
